mercantile/__init__.py

```diff
@@ -326,12 +323,12 @@ def tiles(west, south, east, north, zooms, truncate=False):
             llx = 0 if ll.x < 0 else ll.x
             ury = 0 if ur.y < 0 else ur.y
 
-            for i in range(llx, min(ur.x + 1, 2 ** z)):
-                for j in range(ury, min(ll.y + 1, 2 ** z)):
+            for i in range(llx, min(ur.x + 1, 2**z)):
+                for j in range(ury, min(ll.y + 1, 2**z)):
                     yield Tile(i, j, z)
 
 
-def parent(*tile):
+def parent(*tile, **kwargs):
```

Comment:
Why not `def parent(*tile, zoom=None)`? Does that not work with `*tile`?

Reply:
Keyword arguments can't appear after a variable-length options list in python2 (PEP 3102 is where this change happened for python3), so I had to use **kwargs to capture the zoom argument while still maintaining backwards compatibility (even if it is uglier)